packages/synapse-react-client/src/lib/utils/theme/DefaultTheme.ts

```diff
@@ -79,12 +81,19 @@ const defaultMuiThemeOptions: ThemeOptions = {
         dividers: true,
       },
       styleOverrides: {
-        root: ({ theme }) => ({
+        root: ({ ownerState, theme }) => ({
           color: theme.palette.text.secondary,
-          paddingTop: theme.spacing(3),
-          paddingBottom: theme.spacing(3),
-          paddingLeft: '0px',
-          paddingRight: '0px',
+          paddingTop: ownerState.dividers ? theme.spacing(3) : 0,
+          paddingBottom: ownerState.dividers ? theme.spacing(3) : 0,
+          // Hack - set add a small padding and offset with a negative margin so box-shadow effects on full width elements (like input fields) are shown
+          paddingLeft: DIALOG_INNER_PADDING,
+          paddingRight: DIALOG_INNER_PADDING,
+          marginLeft: `-${DIALOG_INNER_PADDING}`,
+          marginRight: `-${DIALOG_INNER_PADDING}`,
+          // Add a filter on the `dividers` borders so that the borders are transparent on the edges based on the DIALOG_INNER_PADDING length
+          borderImage: ownerState.dividers
+            ? `linear-gradient(to right, transparent 0px, transparent ${DIALOG_INNER_PADDING}, ${theme.palette.grey[400]} ${DIALOG_INNER_PADDING}, ${theme.palette.grey[400]} calc(100% - ${DIALOG_INNER_PADDING}), transparent calc(100% - ${DIALOG_INNER_PADDING})) 1`
+            : undefined,
```

Comment:
Before this change, our input border/box-shadow styles were cut off on the sides if they were full-width in a Dialog, because the DialogContent container had no side padding. These changes

- Add enough side padding to ensure border styles are visible
- Add a counteracting negative margin so the content aligns with the action buttons at the bottom and title at the top.
- Add a border-image filter that basically just hides the first 2px and last 2px of the border, so the border remains visually aligned with the content.